src/GovUk.Education.ExploreEducationStatistics.Common.Tests/Utils/GeographicLevelUtilTests.cs

```diff
@@ -0,0 +1,67 @@
+#nullable enable
+using System.Collections.Generic;
+using System.Linq;
+using GovUk.Education.ExploreEducationStatistics.Common.Model.Data;
+using GovUk.Education.ExploreEducationStatistics.Common.Services;
+using GovUk.Education.ExploreEducationStatistics.Common.Utils;
+using Xunit;
+
+namespace GovUk.Education.ExploreEducationStatistics.Common.Tests.Utils;
+
+public static class GeographicLevelUtilTests
+{
+    public class CsvColumnTests
+    {
+        [Fact]
+        public void CsvColumns()
```

Comment:
Do we have a naming convention for tests?

Maybe `MethodUnderTest_Scenario_Result` ?

Reply:
Not an established one, but typically if the test case name is just the method name, it usually implies the 'default' success case.

Other successful or failure cases are usually covered with extra test cases named in the same way you've suggested.

We'll probably discuss this in a future style guide meeting as it's one of the talking points left on the board.

Reply:
Happy to leave until discussed in style guide meeting :)